### PR TITLE
gpu on prow/bazel take 3: fix some prow config

### PR DESCRIPTION
**Pull request by BenTheElder**
I *think* this will make everything work :upside_down_face:

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -367,12 +367,15 @@ presubmits:
     trigger: "(?m)^/test pull-kubernetes-e2e-gce-gpu-prow,?(\\s+|$)"
     spec:
       containers:
-      - args:
+      - image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170829-93ec4e74
```
Comment (krzyzacy) [inline prow/config.yaml]: keep the latest for now?

Comment (BenTheElder) [reply prow/config.yaml]: I'd like to try and see this job stable enough to convert it to being the default at some point soon, I think `latest` should be reserved for canary jobs.

Comment (krzyzacy) [reply prow/config.yaml]: :confused: then use the up-to-date image, this tag is from bazelbuild I think

Comment (BenTheElder) [reply prow/config.yaml]: Done (I think).

File: prow/config.yaml [hunk 2]
```diff
@@ -367,12 +367,15 @@ presubmits:
...
```
Comment (krzyzacy) [inline prow/config.yaml]: (current version is kubekins-e2e-prow:v20170913-207d52da, and https://github.com/kubernetes/test-infra/pull/4620 will bump it to kubekins-e2e:v20170918-3aa6c113)

Comment (BenTheElder) [reply prow/config.yaml]: bumped to match #4620

**Comment by krzyzacy** [comment]
`TestConfigSecurityJobsMatch` :joy:

**Pull request merged**